### PR TITLE
chore(docs): document how to setup proxy

### PR DESCRIPTION
**Pull request by sranka**
This PR enhances documentation with pointers that help setting up a proxy in communication with InfluxDB.

## Checklist

- [x] CHANGELOG.md updated
- [x] Rebased/mergeable
- [x] `yarn test` completes successfully
- [x] Commit messages are in [semantic format](https://seesparkbox.com/foundry/semantic_commit_messages)

### CODE REVIEW

File: examples/query.ts [hunk 1]
```diff
@@ -1,32 +1,74 @@
 #!./node_modules/.bin/ts-node
```
Comment (sranka) [inline examples/query.ts:1]: @bednar changes in this file seem to be accidentally committed during 1.16 release with dbd20a5cd8d35719b7b9cffaf75f68250132d39e , these are reverted herein ... so it now properly matches the write.js example.

Comment (bednar) [reply examples/query.ts:1]: @sranka like you said, it was a mistake. Sorry for trouble

Comment (sranka) [reply examples/query.ts:1]: ok, I am moving this out from this PR into master

**Review by bednar — APPROVED** [review]
LGTM 👍

**Review by vlastahajek — COMMENTED** [review]
Wouldn't be better to document proxy support in the readme, instead of the link to a comment in an issue?

**Comment by sranka** [comment]
> Wouldn't be better to document proxy support in the readme, instead of the link to a comment in an issue?

No technical details are currently contained in the main Readme. The pointer to a comment is a way to keep the main readme simple.

**Pull request merged**